library/ssl_msg.c

```diff
@@ -2278,6 +2270,27 @@ int mbedtls_ssl_fetch_input( mbedtls_ssl_context *ssl, size_t nb_want )
     return( 0 );
 }
 
+#if defined(MBEDTLS_SSL_USE_MPS)
+int mbedtls_ssl_flush_output( mbedtls_ssl_context *ssl )
+{
+    int ret;
+
+    MBEDTLS_SSL_PROC_CHK( mbedtls_mps_flush( &ssl->mps.l4 ) );
+
+cleanup:
+
+#if defined(MBEDTLS_SSL_USE_MPS)
```

Comment:
nit: This is not needed as L2273 has the same macro.

Reply:
Thanks @lhuang04, fixed!